build/ci/wait_for_k8s_ready.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -xe
+set +o pipefail
+
+is_kubernetes_cluster_ready (){
+  pods=`kubectl get pods -A | awk '{print$3}' | grep -iv ready`
+  all_containers_are_running=false
+  has_not_ready_pod=false
+  for pod in $pods; do
+    running_containers_count=`echo $pod | awk -F / '{print$1}'`
+    total_containers_count=`echo $pod | awk -F / '{print$2}'`
+    if [ $running_containers_count != $total_containers_count ]; then
```

Comment:
please wrap this in a `is_pod_ready` function

Reply:
done